test/abstractmcmc.jl

```diff
@@ -111,6 +112,29 @@ using Statistics: mean
 
     @test m_est_hmc ≈ [49 / 24, 7 / 6] atol = RNDATOL
 
+    samples_sgld = AbstractMCMC.sample(
+        rng,
+        model,
+        hmc,
```

Comment:
```suggestion
        sgld,
```
?

Reply:
Oops, my bad, the previous tests were not actually using the SGLD sampler, now it should be real tests.